style.css

```diff
@@ -40,13 +40,21 @@ header {
   text-decoration: none;
 }
 
+/* Main Section */
+main {
+  width: 390px;
+  height: 6715px;
+  background-color: #fff;
+  padding: 0 24px;
+}
+
 /* Headline Section Styling */
 
-section {
+#headline-container {
   background-image: url("./images/header-illsutration-mobile@2x.png");
   background-repeat: no-repeat;
   background-size: contain;
-  width: 376px;
+  width: 375px;
```

Comment:
- [ Required ] Please consider not giving a fixed width to elements on your web page, at a responsive width of `375px` the webpage introduces a horizontal scroll because the `#headline-container` has a width of `375px` plus `padding` yet the device has a width of `375px`. Please remove this fixed width on the webpage, also see the video above for more details on this.